src/views/Amalgamation/ReviewConfirm.vue

```diff
@@ -237,7 +237,7 @@
         <Certify
           class="py-8 px-6"
           :class="{ 'invalid-section': isCertifyInvalid }"
-          :disableEdit="!isRoleStaff"
+          :disableEdit="false"
```

Comment:
What's the default value of this prop? I'm thinking you could just omit this prop. And if so, which do you think is better: less code, or more verbos code?

Reply:
Thanks for pointing it out. 

About that `disableEdit` prop:

I took a look at the Certify component definition and noticed it's marked as `required: true`. If we skip this prop, we will get this error:
<img width="784" alt="image" src="https://github.com/user-attachments/assets/27737d7a-686e-4a68-87d4-4f68039b5591">

<img width="851" alt="image" src="https://github.com/user-attachments/assets/68dae5ef-7e8b-4d1a-81cb-7a5238bde508">

I've got 2 ideas on how we could handle this:

1. We could stick with what we've got:
It might be a bit wordy, but it makes sure that all the parent components are intentionally setting this value, and we know the current setup works well because it's been tested.

2. Or we could tweak the Certify component and omit this prop in the parent components when we don't need it:
```typescript
// Change this
@Prop({ required: true }) readonly disableEdit!: boolean

// To this
@Prop({ default: false }) readonly disableEdit!: boolean
   ```
We'd need to double-check that it doesn't break anything else

What do you think would work better? I prefer option 2, but we'd need to test it thoroughly. I'd love to hear your thoughts on this.

Reply:
Good analysis.

I assume some thought went into making this property required, so I'm tempted to leave it that way. Thus, option 1 will be fine.

Thanks.

Reply:
Sure, I will update the version number again. Many thanks.